.circleci/config.yml

```diff
@@ -58,6 +63,15 @@ jobs:
             fi
 
 workflows:
+  version: 2
+
+  build-and-linkcheck:
```

Comment:
Instead of two workflows, I think I'd prefer to see this as a single workflow. This workflow is usually called `build` in our repos. You'd just need to combine the `package-and-push-chart-on-tag` and `build-and-linkcheck` jobs into one. Here's what this looks like in the CircleCI UI https://circleci.com/workflow-run/8e10efe4-4379-48a4-9cb4-d5f2296d8490 This keeps the build history cleaner and lets us add more steps as needed.

Reply:
Is that compatible with one of my goals, being that `package-and-push-chart-on-tag` does not trigger the link check again?

Reply:
Yes, as far as I know. You build up a tree structure of jobs that depend on other jobs using `requires`. The package-and-push job and the linkcheck job are independent so you make both of them depend on the build job and your goals should be satisfied. `Require status checks to pass before merging` isn't enabled for this repo so it shouldn't affect merging either.